src/main/kotlin/it/pagopa/wallet/domain/wallets/Wallet.kt

```diff
@@ -75,14 +75,20 @@ data class Wallet(
                 WalletStatusDto.INITIALIZED,
                 WalletStatusDto.VALIDATION_REQUESTED
             )
+        val FINAL_STATUSES =
+            setOf(WalletStatusDto.VALIDATED, WalletStatusDto.ERROR, WalletStatusDto.DELETED)
+    }
+
+    fun isFinalStatus(): Boolean {
+        return FINAL_STATUSES.contains(status)
+    }
+
+    fun isTransientStatus(): Boolean {
```

Comment:
Is this partition safe to maintain? What about to have only one set of status and evaluate final/transient statuses with contains on that set?

Reply:
Set of final statuses and isFinalStatus utility method have been removed since they are no longer needed